lesson03/assignment/data/data.json

```diff
@@ -51,4 +51,31 @@
        "HTML", "CSS", "JavaScript", "Java", "Spring"
     ]
   }
- ]
\ No newline at end of file
+ ];
+
+const employee-info-all = template-hook.map(function(emp) {
```

Comment:
Javascript naming convention is camelCase- no spaces, no hyphens, start with lower case letter then each additional word starts with capital letter.